test/divide_segment.test.js

```diff
@@ -74,8 +74,8 @@ tap.test('divide segments', function(t) {
   });
 
   t.test('possible intersections on 2 polygons', function(t) {
-    var s = subject.geometry.coordinates;
-    var c = clipping.geometry.coordinates;
+    var s = [subject.geometry.coordinates];
+    var c = [clipping.geometry.coordinates];
```

Comment:
The tests enter using the `fillQueue` method rather than the standard `boolean` entry which users use. The first thing that happens in the `boolean` is place everything in an outer array to give us a consistent array structure between polys and multipolys, so Im basically replicating this behaviour. Make sense?